### ISSUE TITLE
Gotta take the long shot

### ISSUE THREAD

**Issue by Nixxen** [ckaotik/Broker_Garbage#12]
Sorry in advance for using the issues tracker for this.

I recently got back to wow through Classic and did my best to remember "that one titan panel broker that let me delete loot based on lowest value". Didn't know the name or how to word the searches but after some hours I eventually found it (well, yeah, I'm here :P).
I see you've stopped playing Wow a long time ago, and probably mostly abandoned this project as you did. There's still a glimmer of hope though. A lot of people came back for classic. Perhaps you're one of them?

I tested the addon in classic, and to no surprise it didn't run smoothly (or at all. In fact it broke some other unrelated addons somehow), and was spewing LUA errors left and right.

In short what I'm trying to get at is; Are you back for classic, and if so, will you be picking up this gem of a timesaver again?

I admit, posting this here is a long shot, but every shot not taken is guaranteed to miss. Or something along those lines. Idioms are not my strong suit.
